WordPress/src/test/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/TagsAndCategoriesUseCaseTest.kt

```diff
@@ -71,7 +74,7 @@ class TagsAndCategoriesUseCaseTest : BaseUnitTest() {
                 listOf(firstTag, secondTag),
                 categoryViews
         )
-        whenever(insightsStore.fetchTags(site, pageSize, forced)).thenReturn(
+        whenever(insightsStore.fetchTags(site, FetchMode.Top(pageSize), forced)).thenReturn(
```

Comment:
Nitpick: We might want to consider renaming "pageSize" to "blockItemCount" to keep it consistent with "BLOCK_ITEM_COUNT" constant from the UseCase. Wdyt?